Hcal/include/Hcal/Event/HcalHit.h

```diff
@@ -111,6 +111,26 @@ class HcalHit : public ldmx::CalorimeterHit {
    */
   int getAmplitudeNeg() const { return amplitudeNeg_; }
 
+  /**
+   * Get the position of the hit
+   * @return position
+   */
+  double getPosition() const { return position_; }
+
+  /**
+   * Get if bar is oriented in X
+   * @return isX
+   */
+  int getIsX() const { return isX_; }
+
+  /**
+   * Get the time difference between the two ends
+   * @return timeDiff
+   */
+  double getTimeDiff() const { return timeDiff_; }
```

Comment:
is timeDiff_ ever set?

Reply:
Not that I know of; but this has been here earlier, I just moved it up to the getters, would you rather have it deleted?

Reply:
It's set in a different producer, so I wont remove it

Reply:
I was going to suggest to add something similar in the HcalRecProducer while you are at it. i.e. to only add it if the hit is in the backHcal. Either way works. I'll approve now